spec/stat_tracker_spec.rb

```diff
@@ -1,6 +1,4 @@
-require 'rspec'
 require 'spec_helper'
-require './lib/stat_tracker'
```

Comment:
why do we no longer need to include stat_tracker?

Reply:
I added those to the spec helper